icd/api/internal_mem_mgr.cpp

```diff
@@ -473,8 +473,12 @@ VkResult InternalMemMgr::CreateMemoryPoolAndSubAllocate(
         Pal::Result palResult = pOwnerList->PushFront(newPool);
         result = PalToVkResult(palResult);
         VK_ASSERT(result == VK_SUCCESS);
```

Comment:
Get rid of this assert since we now handle it not being VK_SUCCESS.

Reply:
Done